cypress/e2e/functional/tile_tests/xy_plot_tool_spec.js

```diff
@@ -137,19 +137,19 @@ context('XYPlot Tool Tile', function () {
       // Added data point will be off the right edge of the plot area until we click 'Fit'.
       xyTile.getTile().scrollIntoView();
       xyTile.getGraphDot().should('have.length', 3);
-      xyTile.getGraphDot().eq(0).should('be.visible');
-      xyTile.getGraphDot().eq(1).should('be.visible');
-      xyTile.getGraphDot().eq(2).should('not.be.visible');
-      // X axis should not have changed in response to adding a data point.
+      xyTile.getGraphDot().eq(0).children('circle.inner-circle').should('be.visible');
+      xyTile.getGraphDot().eq(1).children('circle.inner-circle').should('be.visible');
+      xyTile.getGraphDot().eq(2).children('circle.inner-circle').should('not.be.visible');
```

Comment:
This is a little unclear to me: if 'circle.inner-circle' is selector for a DOM element(s) based on e.g. its class or id, then it ought to be moved down to the relevant "page object" code (actually a "tile object" in our case), /cypress/support/elements/tile/XYPlotToolTile.js; but if its a "code" similar to 'be.visible', then its fine as-is.